img_up/assets/new-prevalence-categories3.csv

```diff
@@ -0,0 +1,52 @@
+IN,69500,"(59,600-79,400)","10,600","(8,300-13,000)","58,900","(49,200-68,500)"
+IA,31400,"(26,800-36,100)","4,400","(3,400-5,400)","27,000","(22,500-31,600)"
+KS,29900,"(25,500-34,300)","4,400","(3,400-5,400)","25,500","(21,200-29,900)"
+KY,49500,"(42,000-57,000)","6,800","(4,900-8,700)","42,700","(35,500-50,000)"
+LA,54900,"(46,600-63,200)","7,900","(6,200-9,700)","47,000","(38,900-55,100)"
+ME,14100,"(11,900-16,300)","1,700","(1,200-2,200)","12,400","(10,300-14,600)"
+MD,59900,"(50,700-69,100)","7,900","(6,200-9,700)","52,000","(42,900-61,000)"
+MA,71600,"(60,900-82,300)","8,400","(6,500-10,300)","63,200","(52,600-73,700)"
+MI,108900,"(93,300-124,500)","13,600","(10,800-16,400)","95,300","(79,900-110,600)"
+MN,53700,"(45,700-61,700)","7,400","(5,900-9,000)","46,300","(38,400-54,100)"
+MS,35700,"(30,600-40,700)","5,100","(3,900-6,300)","30,600","(25,700-35,500)"
+MO,61200,"(52,400-70,000)","8,300","(6,500-10,100)","52,900","(44,200-61,600)"
+MT,10800,"(9,100-12,600)","1,400","(1,000-1,800)","9,400","(7,700-11,100)"
+NE,19600,"(16,600-22,500)","2,800","(2,200-3,500)","16,700","(13,800-19,600)"
+NV,31600,"(26,800-36,400)","4,400","(3,300-5,400)","27,200","(22,500-31,900)"
+NH,13100,"(11,100-15,200)","1,500","(1,100-1,900)","11,600","(9,600-13,700)"
+NJ,92700,"(79,100-106,200)","12,000","(9,500-14,500)","80,600","(67,300-93,900)"
+NM,23200,"(19,800-26,500)","3,400","(2,600-4,200)","19,800","(16,400-23,100)"
+NY,215200,"(186,300-244,000)","26,600","(21,600-31,500)","188,600","(160,200-217,100)"
+NC,110100,"(94,700-125,500)","15,200","(11,800-18,500)","94,900","(79,900-110,000)"
+ND,7300,"(6,200-8,500)","1,000","(700-1,200)","6,400","(5,300-7,500)"
+OH,126400,"(109,300-143,400)","16,900","(13,600-20,300)","109,400","(92,700-126,200)"
+OK,41100,"(34,900-47,300)","6,400","(5,000-7,900)","34,700","(28,700-40,700)"
+OR,42900,"(36,300-49,400)","5,400","(4,100-6,800)","37,400","(31,000-43,900)"
+PA,133000,"(114,600-151,400)","16,900","(13,500-20,200)","116,100","(98,000-134,200)"
+RI,11100,"(9,300-12,900)","1,300","(900-1,700)","9,800","(8,100-11,500)"
+SC,53400,"(45,500-61,300)","7,100","(5,500-8,700)","46,300","(38,500-54,000)"
+SD,8900,"(7,400-10,400)","1,300","(900-1,600)","7,600","(6,200-9,100)"
+TN,73900,"(62,900-84,800)","10,000","(7,800-12,300)","63,800","(53,100-74,600)"
+TX,292900,"(255,400-330,300)","47,200","38,500-56,000)","245,600","(209,200-282,000)"
+UT,29300,"(24,900-33,600)","5,300","(4,100-6,500)","24,000","(19,800-28,200)"
+VT,6300,"(5,300-7,300)",700,(500-900),"5,600","(4,700-6,600)"
+VA,84800,"(72,600-97,000)","11,000","(8,800-13,200)","73,800","(61,800-85,800)"
+WA,74600,"(64,000-85,200)","10,200","(8,100-12,300)","64,400","(54,000-74,800)"
+WV,21500,"(18,100-25,000)","2,500","(1,900-3,100)","19,000","(15,600-22,500)"
+WI,59600,"(50,800-68,300)","7,900","(6,300-9,500)","51,700","(43,100-60,300)"
+WY,5900,"(5,000-6,800)",800,"(600-1,000)","5,100","(4,200-6,000)"

```

Comment:
I'd have to look into this some more, but if you'd like, you can seed your DB with this data. Information on seeding data in Django can be found [here](https://docs.djangoproject.com/en/2.1/howto/initial-data/).